cargo/cargo_build_script.bzl

```diff
@@ -220,10 +238,19 @@ def cargo_build_script(
     cargo_build_script(
         name = "build_script",
         srcs = ["build.rs"],
-        # Data are shipped during execution.
-        data = ["src/lib.rs"],
-        # Environment variables passed during build.rs execution
-        build_script_env = {"CARGO_PKG_VERSION": "0.1.2"},
+        # Optional environment variables passed during build.rs compilation
+        rustc_env = {
+           "CARGO_PKG_VERSION": "0.1.2",
+        },
+        # Optional environment variables passed during build.rs execution.
+        # Note that as the build script's working directory is not execroot,
+        # execpath/location will return an absolute path, instead of a relative
+        # one.
```

Comment:
Does this work under remote-execution? Is it tested by the ci rbe_ubuntu1604 target?

Reply:
Yes, it should work under remote execution - the expansion is done in the cargo_build_script runner within the action.

The build script could opt to do bad things with the path passing it to dependees (though we have some logic in cargo_build_script to try to normalise that too), but within the action this is valid.